.projenrc.js

```diff
@@ -72,48 +69,64 @@ const project = new web.ReactTypeScriptProject({
   autoApproveUpgrades: true,
 });
 
-(function addStorybook() {
-  project.addDevDeps(
-    "@storybook/addon-a11y",
-    "@storybook/addon-actions",
-    "@storybook/addon-essentials",
-    "@storybook/addon-links",
-    "@storybook/addon-storysource",
-    "@storybook/node-logger",
-    "@storybook/preset-create-react-app",
-    "@storybook/react",
-    "babel-loader@8.1.0",
-    "storybook-addon-performance"
-  );
+(function addCypress() {
+  project.addDevDeps("cypress");
 
-  // Add tasks and config for storybook
-  project.addTask("storybook", {
-    exec: "start-storybook -p 6006 -s public",
-    description: "run local storybook server",
+  project.addTask("cypress:open", {
+    exec: "cypress open",
+    description: "open the cypress test runner UI",
   });
 
-  const storybookBuildTask = project.addTask("storybook:build", {
-    exec: "build-storybook -s public",
-    description: "build storybook static site assets",
+  project.addTask("cypress:run", {
+    exec: "cypress run",
+    description: "run the cypress suite in CLI",
   });
-  project.compileTask.prependSpawn(storybookBuildTask);
-
-  project.eslint.addOverride({
-    files: ["**/*.stories.*"],
-    rules: {
-      "import/no-anonymous-default-export": "off",
-      "import/no-extraneous-dependencies": [
-        "error",
+
+  project.gitignore.addPatterns("cypress/videos/", "cypress/screenshots/");
+  project.eslint.addIgnorePattern("cypress/");
+
+  project.buildWorkflow.addJobs({
+    cypress: {
+      runsOn: "ubuntu-latest",
+      permissions: {
+        checks: "write",
+        contents: "read",
+      },
+      steps: [
+        {
+          name: "Checkout",
+          uses: "actions/checkout@v2",
+        },
         {
-          devDependencies: ["**/*.stories.*"],
-          optionalDependencies: false,
-          peerDependencies: true,
+          name: "Cypress Run",
+          uses: "cypress-io/github-action@v2",
+          with: {
+            start: "yarn dev",
+            "wait-on": "http://localhost:3000",
+          },
         },
       ],
     },
   });
+})();
 
-  project.gitignore?.addPatterns("/storybook-static/");
+(function addJest() {
+  project.addDevDeps(
+    // "jest",
+    // "babel-jest",
+    // "ts-node",
+    "@testing-library/react",
+    "@testing-library/jest-dom",
+    "@testing-library/react-hooks",
+    "@testing-library/user-event"
+  );
```

Comment:
This section is in anticipation of moving to next. Currently, we can't install our own version of jest as it will conflict with the jest config that react-app-rewired ships. Once that's out of the way we can uncomment this stuff